OpenTabletDriver.Configurations/Configurations/XP-Pen/XP-Pen Deco 01 V3.json

```
@@ -0,0 +1,33 @@
+{
+  "Name": "XP-Pen Deco 01 V3",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 254,
+      "Height": 158.75,
+      "MaxX": 50800,
+      "MaxY": 31750
+    },
+    "Pen": {
+      "MaxPressure": 16383,
+      "ButtonCount": 2
+    },
+    "AuxiliaryButtons": {
+      "ButtonCount": 8
+    }
+  },
+  "DigitizerIdentifiers": [
+    {
+      "VendorID": 10429,
+      "ProductID": 2375,
+      "InputReportLength": 12,
+      "OutputReportLength": 10,
+      "ReportParser": "OpenTabletDriver.Configurations.Parsers.XP_Pen.XP_PenReportParser",
+      "OutputInitReport": [
+        "ArAE"
+      ]
+    }
+  ],
+  "Attributes": {
+    "Interface": "2"
+  }
```

There is no need to use this attribute when there is no ambiguity in the interface lengths, arbitrarily adding these may prevent new variants of this tablet from being supported out of the box.


```suggestion
  ]
```